lahja/endpoint.py

```diff
@@ -33,62 +41,104 @@
 )
 
 
+class ConnectionConfig(NamedTuple):
+    """
+    Configuration class needed to establish :class:`~lahja.endpoint.Endpoint` connections.
+    """
+    name: str
+    path: pathlib.Path
+
+    @staticmethod
```

Comment:
Why `staticmethod`.  Seems like `classmethod` would be more appropriate since you could use `cls(...)` for the instantiation.

Reply:
Ah yep. Just to make sure I understand the reasoning (part of me is still a Python noob), using `cls`would ensure it works for subclasses as well, right?